Octokit.Tests/Clients/MiscellaneousClientTests.cs

```diff
@@ -58,6 +59,72 @@ public class TheGetEmojisMethod
             }
         }
 
+        public class TheGetResourceRateLimitsMethod
+        {
+            [Fact]
+            public async Task RequestsTheRecourceRateLimitEndpoint()
+            {
+                IApiResponse<MiscellaneousRateLimit> response = new ApiResponse<MiscellaneousRateLimit>
+                (
+                    new Response(),
+                    new MiscellaneousRateLimit(
+                        new ResourceRateLimit(
+                            new RateLimit(5000, 4999, 1372700873),
+                            new RateLimit(30, 18, 1372700873)
+                        ),
+                        new RateLimit(100, 75, 1372700873)
+                    )
+                );
+                var connection = Substitute.For<IConnection>();
+                connection.Get<MiscellaneousRateLimit>(Args.Uri, null, null).Returns(Task.FromResult(response));
+                var client = new MiscellaneousClient(connection);
+
+                var result = await client.GetRateLimits();
+
+                // Test the high level object
+                Assert.NotNull(result);
```

Comment:
While I love the explicitness here, I think letting this fall through to the Assert.Equals is totally fine to simplify the tests...